WORKSPACE

```diff
@@ -20,7 +20,8 @@ load("//tensorflow:workspace.bzl", "tf_workspace")
 #android_sdk_repository(
 #    name = "androidsdk",
 #    api_level = 23,
-#    build_tools_version = "25.0.1",
+#    #Ensure that your build_tools_version (below) the SDK manager version as it updates periodicaly.
```

Comment:
\# Ensure that you have the build_tools_version below installed in the SDK manager as it updates periodically.